todo_app/templates/index.html

```diff
@@ -8,6 +8,26 @@ <h1 class="display-4">To-Do App</h1>
   </div>
 
   <form action="add_todo" method="post">
+    <label for="name>"Enter New Todo: </label>
+    <input type="text" name="title" required />
+    <input type="submit" value="Add New Todo" />
+  </form>
+
+  <div class="row">
```

Comment:
Is there a reason you've added all of this code a second time, since I'm seeing your to-dos are listed twice?